contrib/sync_namespaces_jobs.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+import os
+
+import yaml
+
+DST_DIR = "/tmp/tron-servdir"
+#DST_DIR = "/nail/home/chl/tron-servdir"
+SRC_DIR = "/nail/etc/services/tron/prod"
+
+#SRC_DIR = "/nail/home/chl/git/yelpsoa-configs/tron/prod"
+#DST_DIR = "/nail/home/chl/Tron/contrib/tron-servdir"
+
+
+def main():
+    print(
+        "This script will load the same amount of namepance and jobs from tron-prod to tron-playground."
```

Comment:
Seems like this script doesn't do any loading, it just copies and modifies some files from SRC_DIR to DST_DIR. Probably don't need the warning? And this note (or the docstring) should describe what the script does better.